src/systemctl/systemctl-show.c

```diff
@@ -821,18 +821,33 @@ static void print_status_info(
 }
 
 static void show_unit_help(UnitStatusInfo *i) {
+        int r;
+
         assert(i);
 
         if (!i->documentation) {
                 log_info("Documentation for %s not known.", i->id);
                 return;
         }
 
-        STRV_FOREACH(p, i->documentation)
-                if (startswith(*p, "man:"))
-                        show_man_page(*p + 4, false);
-                else
-                        log_info("Can't show: %s", *p);
+        STRV_FOREACH(doc, i->documentation) {
+                const char *p;
+
+                if (doc != i->documentation) {
+                        puts("");
+                        fflush(stdout);
+                }
+
+                if ((p = startswith(*doc, "man:")))
+                        show_man_page(p, /* null_stdio= */ false);
+                else if ((p = startswith(*doc, "file://"))) {
```

Comment:
this is just wrong. if packages ship html docs in /usr/share/docs/ then we should certainly not dump the raw html on stdout.

sorry, but I am not convinced this makes any sense at all.

Reply:
I see your point in this comment @poettering, and have to unfortunately agree with you :rofl: 

So yea it needs an extra step to check the mime-type of the referenced file.
If it aint `text/plain` it should maybe fallback to using the `http://` / `https://` branch in the code, but instead of using those URI's using `file://` instead. :thinking: